### DISCUSSION TITLE
Overriding query

### DISCUSSION THREAD

**Discussion by KokhanevychD** [python-gino/gino#758] [Q&A]
HI! Is there a way in Gino models to override query, in the way you can create custom manager in django orm models?

**Comment by wwwjfy** [comment]
Could you elaborate? What do you mean by overriding?

**Reply by KokhanevychD** [reply]
I mean a situation when we need to change query all over the code, for example, to select only instances with some bool field sate, can we override basic query on a model level with gino or no?
In django orm we can use custom [managers](https://docs.djangoproject.com/en/3.1/topics/db/managers/), so I'm looking for something like this
